File: app/src/main/java/com/kelsos/mbrc/data/db/NowPlayingDatabase.java
package com.kelsos.mbrc.data.db;

import com.raizlabs.android.dbflow.annotation.Database;
@Database(version = NowPlayingDatabase.VERSION, name = NowPlayingDatabase.NAME)
public class NowPlayingDatabase {
  static final int VERSION = 1;
  static final String NAME = "now_playing";
}

File: app/src/main/kotlin/com/kelsos/mbrc/data/db/NowPlayingDatabase.kt
package com.kelsos.mbrc.data.db

import com.raizlabs.android.dbflow.annotation.Database

@Database(version = NowPlayingDatabase.VERSION, name = NowPlayingDatabase.NAME)
object NowPlayingDatabase {
    const val VERSION = 1
    const val NAME = "now_playing_data"
}
